src/plugins/data_source/server/client/client_config.test.ts

```diff
@@ -17,9 +18,13 @@ describe('parseClientOptions', () => {
       clientPool: {
         size: 5,
       },
+      globalOpenSearchConfig: {
+        requestTimeout: duration(1, 'seconds'),
+        pingTimeout: duration(2, 'seconds'),
+      },
     } as DataSourcePluginConfigType;
 
-    expect(parseClientOptions(config, TEST_DATA_SOURCE_ENDPOINT)).toEqual(
+    expect(parseClientOptions(config, TEST_DATA_SOURCE_ENDPOINT, [])).toEqual(
       expect.objectContaining({
         node: TEST_DATA_SOURCE_ENDPOINT,
         ssl: {
```

Comment:
can we verify `requestTimeout` and `pingTimeout` too?

Reply:
Sure, added yet, thank you!